source/index.txt

```diff
@@ -198,4 +202,4 @@ Take the following free online courses taught by MongoDB instructors:
      
      - `MongoDB Node.js Developer Path <https://learn.mongodb.com/learning-paths/mongodb-nodejs-developer-path>`_
         
-       Learn how to build a Node.js application using MongoDB as your database.
+       Learn how to build a Node.js application that uses MongoDB as a database.
```

Comment:
Question:
From reading the description of each course, I wasn't able to determine a clear difference between the two courses. If possible, could the descriptions be updated so readers can determine which one might be more suitable?